proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -871,39 +898,39 @@ ProToken directive() throws IOException {
         getChar();
       }
       macroUndefine();
-      return makeToken(ProParserTokenTypes.AMPUNDEFINE);
+      return makeToken(PreprocessorParser.AMPUNDEFINE);
     }
 
     if (macroType.equals("&analyze-suspend")) {
       appendToEOL();
       getChar();
-      return makeToken(ProParserTokenTypes.AMPANALYZESUSPEND);
+      return makeToken(PreprocessorParser.AMPANALYZESUSPEND);
     }
     if (macroType.equals("&analyze-resume")) {
       appendToEOL();
       getChar();
-      return makeToken(ProParserTokenTypes.AMPANALYZERESUME);
+      return makeToken(PreprocessorParser.AMPANALYZERESUME);
     }
     if (macroType.equals("&message")) {
       appendToEOL();
       getChar();
-      return makeToken(ProParserTokenTypes.AMPMESSAGE);
+      return makeToken(PreprocessorParser.AMPMESSAGE);
     }
 
     if (macroType.equals("&if")) {
-      return makeToken(ProParserTokenTypes.AMPIF);
+      return makeToken(PreprocessorParser.AMPIF);
     }
     if (macroType.equals("&then")) {
-      return makeToken(ProParserTokenTypes.AMPTHEN);
+      return makeToken(PreprocessorParser.AMPTHEN);
     }
     if (macroType.equals("&elseif")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "&elseif" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1132)